lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -612,7 +613,12 @@ module.exports = class MethodMappings {
               throw err;
             }
           }
-          await element.sendKeys(value);
+          if (redacted) {
+            // Using Key.NULL as a flag for redacted values.
+            await element.sendKeys(value, Key.NULL);
+          } else {
+            await element.sendKeys(value);
+          }
```

Comment:
Move this logic to `setElementValueRedacted` only (modify the `value` there only). You can also replace `...args` in that function with `webElementOrId, value`.

Reply:
done  73484d5